pmm-app/tests/ia/pages/api/templatesAPI.js

```diff
@@ -0,0 +1,45 @@
+const { I, ruleTemplatesPage } = inject();
+const assert = require('assert');
+
+module.exports = {
+  async createRuleTemplate(path) {
+    const [, content, id] = await ruleTemplatesPage.ruleTemplate.templateNameAndContent(path);
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+    const body = {
+      yaml: content,
+    };
+    const resp = await I.sendPostRequest('v1/management/ia/Templates/Create', body, headers);
+
+    assert.ok(resp.status === 200, `Failed to create a template with id "${id}".
+     Response message is "${resp.data.message}"`);
+
+    return id;
+  },
+
+  async clearAllTemplates() {
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+    const resp = await I.sendPostRequest('v1/management/ia/Templates/List', { reload: true }, headers);
+
+    if (resp.data === {}) return;
```

Comment:
This will never be true, since the object `{}` gets re-created on every pass. In this case we should check like this `Object.keys(resp.data).length === 0` - this is reliable.

Reply:
done, thanks